pages/api/index.js

```diff
@@ -10,6 +12,14 @@ function API({url, content, locale}) {
       locale={locale}
       contributors={content.contributors}
     >
+      <Head>
+        {twitterCard({
+          card: 'summary',
+          title: content.title,
```

Comment:
I get get it done with `glamorous - Basic Concepts` but `Basic Concepts | glamorous` looks fine too. Whichever that you feel the best for it, I will implement it that way 😄